### PR TITLE
chore: configure eqeqeq lint rule

### PR DESCRIPTION
**Pull request by Shinigami92**
Related to https://github.com/faker-js/faker/pull/589#discussion_r819427373

### CODE REVIEW

File: test/address.spec.ts [hunk 1]
```diff
@@ -622,8 +622,8 @@ describe('address', () => {
           expect(coordinate[1]).toBeTypeOf('string');
 
           const distanceToTarget =
-            Math.pow(coordinate[0] - latitude, 2) +
-            Math.pow(coordinate[1] - longitude, 2);
+            Math.pow(+coordinate[0] - latitude, 2) +
+            Math.pow(+coordinate[1] - longitude, 2);
```
Comment (ST-DDT) [inline test/address.spec.ts:626]: Is this change related?

Comment (Shinigami92) [reply test/address.spec.ts:626]: Rebase after #597 merged

**Review by prisis — APPROVED** [review]

**Review by ST-DDT — APPROVED** [review]
